base/promotion.jl

```diff
@@ -333,7 +333,9 @@ If `y` is an `Int` literal (e.g. `2` in `x^2` or `-3` in `x^-3`), the Julia code
 enable compile-time specialization on the value of the exponent.
 (As a default fallback we have `Base.literal_pow(^, x, Val(y)) = ^(x,y)`,
 where usually `^ == Base.^` unless `^` has been defined in the calling
-namespace.)
+namespace.) If `y` is a negative integer literal, then `Base.literal_pow`
+transforms the operation to `inv(x)^y` by default.
```

Comment:
This seems incorrect. Should it not say `inv(x)^-y`?

Reply:
We are taking the inverse of x because of the negative exponent. x^-1 = inv(x), hence x^-y = inv(x)^y

Reply:
yes, technically it should be the absolute value of y. Should I open a new issue and fix it?

Reply:
If `y` is negative then `-y` is correct, no? Yes please, it would be great to have PR to fix this.

Reply:
ah good point, yes please open a PR.